connect/runtime/src/main/java/org/apache/kafka/connect/runtime/AbstractHerder.java

```diff
@@ -1082,12 +1081,8 @@ protected final boolean maybeAddConfigErrors(
 
     private String trace(Throwable t) {
         ByteArrayOutputStream output = new ByteArrayOutputStream();
-        try {
-            t.printStackTrace(new PrintStream(output, false, StandardCharsets.UTF_8.name()));
-            return output.toString(StandardCharsets.UTF_8.name());
-        } catch (UnsupportedEncodingException e) {
-            return null;
-        }
+        t.printStackTrace(new PrintStream(output, false, StandardCharsets.UTF_8));
+        return output.toString(StandardCharsets.UTF_8);
```

Comment:
nice cleanup. open https://issues.apache.org/jira/browse/KAFKA-19137 to cleanup remaining usages.